src/Container.js

```diff
@@ -66,6 +66,19 @@ export default class Container {
         this._registrations[name] = registration;
         return new RegistrationModifier(registration, this._instanceCache, this._registrationGroups);
     }
+    registerFactory(name, externallyOwnedFactory) {
```

Comment:
I think this can just be called `factory`. The lifetime scope isn't somthing configured or inferered at this point.